File: U1/TI1.3 RESTFULL_JAVA_SINBDD_GR01/02 CLIWEB/ConUni_Rest_Web_GR01/src/main/java/ec/edu/monster/model/ResultadoConversion.java
package ec.edu.monster.model;

/**
 * Modelo para los datos de resultado de conversión
 * @author YourName
 */
public class ResultadoConversion {
    private double valorOriginal;
    private double valorConvertidoExacto;
    private double valorConvertidoRedondeado;
    private String unidadOrigen;
    private String unidadDestino;
    private String tipoConversion;
    private double factorConversion;
    private String fechaConversion;
    
    // Getters y Setters
    public double getValorOriginal() {
        return valorOriginal;
    }
    
    public void setValorOriginal(double valorOriginal) {
        this.valorOriginal = valorOriginal;
    }
    
    public double getValorConvertidoExacto() {
        return valorConvertidoExacto;
    }
    
    public void setValorConvertidoExacto(double valorConvertidoExacto) {
        this.valorConvertidoExacto = valorConvertidoExacto;
    }
    
    public double getValorConvertidoRedondeado() {
        return valorConvertidoRedondeado;
    }
    
    public void setValorConvertidoRedondeado(double valorConvertidoRedondeado) {
        this.valorConvertidoRedondeado = valorConvertidoRedondeado;
    }
    
    public String getUnidadOrigen() {
        return unidadOrigen;
    }
    
    public void setUnidadOrigen(String unidadOrigen) {
        this.unidadOrigen = unidadOrigen;
    }
    
    public String getUnidadDestino() {
        return unidadDestino;
    }
    
    public void setUnidadDestino(String unidadDestino) {
        this.unidadDestino = unidadDestino;
    }
    
    public String getTipoConversion() {
        return tipoConversion;
    }
    
    public void setTipoConversion(String tipoConversion) {
        this.tipoConversion = tipoConversion;
    }
    
    public double getFactorConversion() {
        return factorConversion;
    }
    
    public void setFactorConversion(double factorConversion) {
        this.factorConversion = factorConversion;
    }
    
    public String getFechaConversion() {
        return fechaConversion;
    }
    
    public void setFechaConversion(String fechaConversion) {
        this.fechaConversion = fechaConversion;
    }
}

File: U1/TI1.4 RESTFULL_DOTNET_SINBDD_GR01/02CLIWEB/ConUni_CLIWEB_Rest/ec/edu/monster/models/ResultadoConversion.cs
using System;

namespace ConUni_CLIWEB_Rest.ec.edu.monster.models
{
    /// <summary>
    /// Modelo para los datos de resultado de conversión
    /// </summary>
    public class ResultadoConversion
    {
        public double ValorOriginal { get; set; }
        public double ValorConvertidoExacto { get; set; }
        public double ValorConvertidoRedondeado { get; set; }
        public string UnidadOrigen { get; set; }
        public string UnidadDestino { get; set; }
        public string TipoConversion { get; set; }
        public double FactorConversion { get; set; }
        public DateTime FechaConversion { get; set; }

        public ResultadoConversion()
        {
            UnidadOrigen = string.Empty;
            UnidadDestino = string.Empty;
            TipoConversion = string.Empty;
            FechaConversion = DateTime.Now;
        }
    }
}